finish/src/main/java/io/openliberty/guides/circuitbreaker/global/eBank/servlets/CircuitBreakerServlet.java

```diff
@@ -62,17 +62,17 @@ protected void doGet(HttpServletRequest request,
 
         response.setContentType("text/html");    
 
-        responseMsg += "<html><body><center><h1>Global eBank</h1></center><hr>";
+        //responseMsg += "<html><body><center><h1>Global eBank</h1></center><hr>";
```

Comment:
Why leave in all this commented out code?

Reply:
Agreed and fixed.